packages/spright-components/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Spright Components
+
+This package generally uses the same tooling and policies as Nimble components. See the [`nimble-components` CONTRIBUTING doc](../nimble-components/CONTRIBUTING.md) for an overview. For details on whether a component belongs in Nimble or Spright, refer to the [README](./README.md). This document describes ways in which Spright development differs from Nimble.
```

Comment:
Paths in markdown need to be relative to repo root
`/packages/nimble-components/CONTRIBUTING.md`
`/packages/spright-components/README.md`